plugins/catalog/src/components/useOwnUser.ts

```diff
@@ -21,20 +21,21 @@ import { identityApiRef, useApi } from '@backstage/core';
 import { catalogApiRef } from '../plugin';
 
 /**
- * Get the group memberships of the logged-in user.
+ * Get the catalog User entity (if any) that matches the logged-in user.
  */
-export function useOwnUser(): AsyncState<UserEntity> {
+export function useOwnUser(): AsyncState<UserEntity | undefined> {
```

Comment:
this is not a functional change nor used outside of the package, so needs no changeset